src/navigation/stack/mainBucketStack.js

```diff
@@ -1,5 +1,5 @@
 import { StackNavigator, DrawerNavigator, DrawerItems } from 'react-navigation';
-import { View, StyleSheet } from 'react-native';
+import { View, StyleSheet, BackHandler } from 'react-native';
```

Comment:
BackHandler is defined and never used. Is it needed?

Reply:
its not needed here.